### PR TITLE
Feature/currency comparison

### PR DESCRIPTION
**Pull request by ckarczewski**

### CODE REVIEW

File: backend/nbp/views.py [hunk 1]
```diff
@@ -108,4 +110,59 @@ def session_scheduling(request):
...
+
+                    data_array.clear()
+
+            res.append(schedule_currency_pairs(data_array_C1, data_array_C2, time))
+
+        
+    return JsonResponse(res, safe=False)
+
+'''
+1 2 3 4 5
```
Comment (jowczar) [inline backend/nbp/views.py:164]: what is this stuff?

Comment (ckarczewski) [reply backend/nbp/views.py:164]: comment I forgot to delete

File: backend/nbp/urls.py [hunk 2]
```diff
@@ -5,4 +5,5 @@
     path('measures', views.measures, name='measures'),
     path('currency_list', views.currency_list, name='currency_list'),
     path('session', views.session_scheduling, name='session_scheduling'),
+    path('currency_comparison', views.currency_comparison, name='currency comparison'),
```
Comment (ShanghaiG) [inline backend/nbp/urls.py:8]: that space inside currency comparision is a proper way to name path? Will it work ?

File: backend/nbp/views.py [hunk 3]
```diff
@@ -108,4 +110,52 @@ def session_scheduling(request):
...
+        data_array_C2 = []
+        data_array = []
+        res = []
+
+        # find code in database
+        for i in range(2):
+            time = time_intervals[i]
+            for i in range(2):
+                query_A = TabelaA.objects.values_list('mid').filter(code=currency[i]).filter(country=currency[i])
+                query_B = TabelaB.objects.values_list('mid').filter(code=currency[i]).filter(country=currency[i])
```
Comment (ShanghaiG) [inline backend/nbp/views.py:134]: Wouldnt it be better if search in TableA first, if found in TableA then DONT search in TableB?
What is the point in looping over TableB if data found in TableA?

File: backend/nbp/views.py [hunk 4]
```diff
@@ -1,8 +1,10 @@
 from django.shortcuts import (HttpResponse)
 from django.http import JsonResponse
 from nbp.helpers.statistical_measures import median, dominant, standard_deviation, coefficient_of_variation
+from nbp.helpers.currency_change_schedule import schedule_currency_pairs
 from datetime import datetime, timedelta
 from nbp.models import TabelaA, TabelaB, TabelaC
+
```
Comment (ShanghaiG) [inline backend/nbp/views.py:7]: I think, TabelaA, TabelaB, TabelaC should be named in english as well

**Review by ShanghaiG — CHANGES_REQUESTED** [review]

File: backend/nbp/views.py (hunk 3 above)
Comment (ckarczewski) [reply backend/nbp/views.py:134]: no

File: backend/nbp/views.py (hunk 4 above)
Comment (ckarczewski) [reply backend/nbp/views.py:7]: table names are imported from the database

**Comment by jowczar** [comment]
answer comments

**Review by jowczar — CHANGES_REQUESTED** [review]
answer comments

File: backend/nbp/urls.py (hunk 2 above)
Comment (ckarczewski) [reply backend/nbp/urls.py]: Probably my oversight, already fixed

File: backend/nbp/views.py (hunk 3 above)
Comment (ShanghaiG) [reply backend/nbp/views.py:134]: ok, thanks for answering!

**Review by ShanghaiG — APPROVED** [review]

**Review by jowczar — APPROVED** [review]

**Pull request merged**